tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogExtensions.cs

```diff
@@ -16,10 +19,23 @@ public static class DatadogExtensions
     /// Configure the Datadog Exporter, Diagnoser and Column hiding rule
     /// </summary>
     /// <param name="config">Configuration instance</param>
+    /// <param name="enableProfiler">True to enable Datadog's Profiler; a null value will parse DD_PROFILING_ENABLED environment variable</param>
     /// <returns>Same configuration instance</returns>
-    public static IConfig WithDatadog(this IConfig config)
+    public static IConfig WithDatadog(this IConfig config, bool? enableProfiler = null)
```

Comment:
Breaking change alert 😛 IMO should be 
```suggestion
    public static IConfig WithDatadog(this IConfig config)
      => config.WithDatadog(enableProfiler: null);

    public static IConfig WithDatadog(this IConfig config, bool? enableProfiler)
```

But as you've already pointed out noone else uses it, you can have whatever you prefer 😛 Optional parameters aren't as easily extended later though for public APIs

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4650/commits/7e6a2da8f95126dd7ae290f6c034c3f87b0a251d